lms/models/oauth_state.py

```diff
@@ -13,13 +16,10 @@ class OauthState(BASE):
     lti_params = sa.Column(sa.String)
 
 
-def find_by_state(session, state):
-    return session.query(OauthState).filter(
-        OauthState.guid == state).one_or_none()
```

Comment:
`find_by_state()` has been moved to a private func below

Reply:
These comments are priceless-ly helpful. Thanks!